source-control/git.md

```diff
@@ -211,8 +211,77 @@ git stash
 git stash pop
 ```
 
-Or you can simply move the current state into a new branch:
+Or you can move the current state into a new branch:
 
 ```bash
 git stash branch <new_branch_to_save_changes>
 ```
+
+### Working with images, video and other binary content
+
+Avoid committing frequently changed binary files, such as large images, video or compiled code to your git repository. Binary content is not diffed like text content, so the repository will take longer and longer to clone.
+
+One solution to this problem is `Git LFS (Git Large File Storage)` - an open source Git extension for versioning large files. `Git LFS` replaces the binary files with text pointers inside Git, while storing the file contents on a remote server like GitHub.com or Azure DevOps.
+
+#### Benefits of Git LFS
+
+* Uses the end to end Git workflow for all files
+* Binary files tracked by Git LFS can be as big as you need them to be
```

Comment:
What does this statement mean? There is no upper limit on the size of files tracked by Git LFS? Not sure that is true: https://github.com/git-lfs/git-lfs/wiki/Limitations

Reply:
I'll remove that statement, I believe you're right